docs/source/markdown/options/interactive.md

```diff
@@ -4,4 +4,8 @@
 ####> are applicable to all of those.
 #### **--interactive**, **-i**
 
-When set to **true**, keep stdin open even if not attached. The default is **false**.
+When set to **true**, make stdin available to the contained process. If **false**, the stdin of the contained process is empty and immediately closed.
+
+If attached, stdin is piped to the contained process. If detached, reading stdin will block until later attached.
+
+**Caveat:** podman will consume input from stdin as soon as it becomes available, even if the contained process doesn't request it. This is a design limitation derived from piping stdio over sockets and not exclusive to podman.
```

Comment:
capitalize Podman and I dont think the second sentence adds value here.

Reply:
> I dont think the second sentence adds value here

What part? If you're referring to the entire Caveat, it may seem obvious if you think about how podman works internally, but as a user it is not that obvious. At the very least it wasn't obvious to me and neither it was obvious to the person who wrote the code I was trying to fix.

Reply:
"This is a design limitation ..."  For man pages, it seems out of place.

Reply:
Updated accordingly.